frontend/packages/integration-tests-cypress/tests/crud/other-routes.spec.ts

```diff
@@ -21,7 +21,12 @@ describe('Visiting other routes', () => {
       path: '/',
       waitFor: () => {
         cy.byLegacyTestID('resource-title').should('exist');
+        cy.byTestID('loading-indicator').should('not.exist');
         cy.byTestID('skeleton-chart').should('not.exist');
+        // redirects to Overview dashboard where many cards load asynchronously
+        // cannot determine a 'dashboard completely loaded' state, so waiting
+        // eslint-disable-next-line cypress/no-unnecessary-waiting
+        cy.wait(4000); // wait 4 seconds
```

Comment:
I'd rather not put in an arbitrary wait... I think it's better to wait for some particular content to show up (even if it means not all the cards have data).

Reply:
Removed the wait.  I believe the missing piece was checking for the loading indicator before the chart skeleton.